pkg/skaffold/deploy/deploy_mux.go

```diff
@@ -99,33 +99,30 @@ func (m DeployerMux) RegisterLocalImages(images []graph.Artifact) {
 	}
 }
 
-func (m DeployerMux) Deploy(ctx context.Context, w io.Writer, as []graph.Artifact) ([]string, error) {
-	seenNamespaces := util.NewStringSet()
-
+func (m DeployerMux) Deploy(ctx context.Context, w io.Writer, as []graph.Artifact) error {
 	for i, deployer := range m.deployers {
 		eventV2.DeployInProgress(i)
 		w = output.WithEventContext(w, constants.Deploy, strconv.Itoa(i), "skaffold")
 		ctx, endTrace := instrumentation.StartTrace(ctx, "Deploy")
 
-		namespaces, err := deployer.Deploy(ctx, w, as)
-		if err != nil {
+		var err error
```

Comment:
nit: any reason to not just use `:=` on 109 and 115?

Reply:
nope I should have done that, updated